src/commands/run.rs

```diff
@@ -253,15 +250,14 @@ mod test {
     }
 }
 
-#[derive(Debug, Fail)]
+#[derive(Debug, Error)]
 enum RunError {
-    #[fail(display = "Failed to run command \"{}\". {}", _0, _1)]
+    #[error("Failed to run command \"{0}\". {1}")]
     CannotRegenLockfile(String, dataflow::Error),
-    #[fail(
-        display = "The command \"{}\" for module \"{}\" is defined but the source at \"{}\" does not exist.",
-        _0, _1, _2
+    #[error(
+        "The command \"{0}\" for module \"{1}\" is defined but the source at \"{2}\" does not exist.",
     )]
     SourceForCommandNotFound(String, String, String),
-    #[fail(display = "Failed to run {}: {}", runtime, error)]
-    ProcessFailed { runtime: String, error: String },
+    #[error("Failed to run {0}: {1}")]
+    ProcessFailed(String, String),
```

Comment:
Minor nit, the fields here help users receiving this error to know what the strings they're looking at are, otherwise it's not obvious